playground/frontend/assets/thumb_up.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
just please add license headers

Reply:
@pabloem I don't see a RAT check result for the PR, do I need to trigger it somehow?